lib/fetcher.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const co = require('co')
+
+function * addon (heroku, app, db) {
+  const {resolve} = require('heroku-cli-addons')
+  let attachment = yield resolve.attachment(heroku, app, db)
+  return yield heroku.get(`/addons/${attachment.addon.id}`)
+}
+
+function * all (heroku, app) {
+  const sortBy = require('lodash.sortby')
+  const notDatabaseUrl = (a) => !a.config_vars.find((c) => c === 'DATABASE_URL')
```

Comment:
Maybe out of scope here, but should we have a secondary sort order by addon name or attachment name or something?